restapi/src/main/java/io/stargate/web/docsapi/service/util/DocsApiUtils.java

```diff
@@ -61,6 +62,37 @@ public static String convertArrayPath(String path) {
     }
   }
 
+  /**
+   * Converts any literal unicode points into their actual character. E.g. if the input string is
+   * literally abc\u002E123, this function returns abc.123 This allows a user to use unicode escape
+   * sequence in where filters when it would otherwise be ambiguous to use the corresponding
+   * character. Any path that already has square brackets in it will be left alone.
```

Comment:
So this skips bracketed-only, like "[1234]"?

Reply:
Yes, that's correct

Reply:
I should have asked: why is this needed/intended?

Reply:
AFAIK this is to avoid clashes with indexes of JSON array elements when they are stored in rows.